Development.md

```diff
@@ -277,6 +277,12 @@ $ ci/cleanup.sh
 
 This generally uses a specific version number (e.g. 20160323164322) that has been released but not published. Passing integration tests offer confidence that the release can be published.
 
+## Testing
+
+* To run the test suite locally, run `make test`
+* To run a subset of tests, provide `TEST`, e.g. `make test TEST=github.com/convox/rack/cmd/convox`
+* To run a single test, provide `TEST` and `TESTRUN`, e.g. `make test TEST=github.com/convox/rack/cmd/convox TESTRUN=TestRequiredFlagsWhenInstallingIntoExistingVPC`
```

Comment:
Any thoughts on renaming:
TEST -> PKG (this would indicate its the golang package the test belongs too)
TESTRUN -> TEST (actual test to run)